lbrynet/wallet/manager.py

```diff
@@ -54,19 +55,65 @@ def network(self):
 
     @property
     def use_encryption(self):
-        # TODO: implement this
-        return False
+        return self.default_account.serialize_encrypted
 
     @property
     def is_first_run(self):
         return True
```

Comment:
Do we still need `is_first_run`? I know it's not part of this PR, but just wondering if we can drop it.